src/MCPClient/lib/clientScripts/restructureDIPForContentDMUpload.py

```diff
@@ -640,569 +144,115 @@ def addAipUuidToDcMetadata(dipUuid, dcMetadata):
-        if len(dcMetadata['title']) > 1:
-            titleValues += '; '.join(dcMetadata['title'])
-    fullFileContents = generateFullFileEntry(titleValues, 'index', '.cpd')
-
-    # Archivematica's structMap is always the first one; the user-submitted
-    # structMap (if it exists) is always the second one. If the user-submitted
-    # structMap is present, parse it for the SIP structure so we can use that
-    # structure in the CONTENTdm packages.
-    if (len(structMaps)) == 2:
-        structMapDom = structMaps[1]
-    else:
-        structMapDom = structMaps[0]
-    structMapDict = parseStructMap(structMapDom, filesInObjectDirectoryForThisDmdSecGroup)
-
-    # Determine the order in which we will add the child-level rows to the .cpd and .full files.
-    Orders = []
-    for fptr, details in structMapDict.iteritems():
-        Orders.append(details['order'])
-
-    # Iterate through the list of order values and add the matching structMapDict entry
-    # to the .cpd file (and copy the file into the output directory).
-    for order in sorted(Orders):
-        for k, v in structMapDict.iteritems():
-            # Get each access file's base filesname without extension, since we'll use it
-            # for the .icon and .desc files.
-            accessFileBasenameName, accessFileBasenameExt = os.path.splitext(v['filename'])
-            # The UUID is the first 36-characters of the filename.
-            accessFileUuid = accessFileBasenameName[:36]
-            # Remove the UUID from the basename.
-            accessFileBasenameName = accessFileBasenameName[37:]
-            # Reassemble the basename.
-            accessFileBasenameName = accessFileBasenameName + '-' + accessFileUuid
-
-            # Get the name of the first file in the sorted order; we use this later to create
-            # a thumbnail for current parent item.
-            if v['order'] == '00001':
-                parentThumbnailFilename = accessFileBasenameName + '.icon' 
+def generate_project_client_package(output_dir, package_type, structmap, dmdsecs, dipuuid):
+    """
+    Generates a simple.txt or compound.txt from the METS of a DIP
 
-            if order == v['order']:
-                # Process each object file.
-                for fullPath in filesInObjectDirectoryForThisDmdSecGroup:
-                    # For each object file, output the object file. We need to find the full path
-                    # of the file identified in v['filename'].
-                    if (v['filename'] in fullPath):
-                        shutil.copy(fullPath, os.path.join(outputItemDir, accessFileBasenameName + accessFileBasenameExt))
+    :param output_dir: Path to directory for simple/compound.txt
+    :param structmap: structMap element from the METS (Preparse somehow?)
+    :param dmdsecs: Dict of {<DMDID>: OrderedDict{column name: value} or <dmdSec element>? }
+    :param dipuuid: UUID of the DIP
+    """
+    print 'dipuuid', dipuuid
```

Comment:
Yeah, I'm glad to have logging - I'd just rather it be easy to understand. If the output were a bit clearer/more verbose I think this'd work for me :+1: